index.css

```diff
@@ -0,0 +1,53 @@
+/* Importing All Components CSS */
+@import "/components.css";
+
+/* Keyframes for Load Animation */
+
```

Comment:
Since all the component files have no empty line between a comment and class, I suggest that the same can be followed here to maintain consistency.

Reply:
done. thank you for the suggestions.